src/forms/Form/Form.stories.tsx

```diff
@@ -143,6 +143,7 @@ export const Playground = (): ReactElement => {
 	const prepopulateValues = object("Prepolulate values", {
 		"textField": "Text field prepopulated",
 		"textArea": "Text area prepopulated",
+		"date": new Date(),

```

Comment:
I think the Date issue might still be present, here is what the date looks like on the form state 
![image](https://user-images.githubusercontent.com/23409284/169586308-4b5e87e9-f984-442a-819e-ffb528e37358.png)

and this is the hour on the field
![image](https://user-images.githubusercontent.com/23409284/169586377-1321857b-87f5-4915-a3ef-de327752ef42.png)

It says it should be 17:56 pm but since we're on a GMT-5 time zone the actual component says 12:56, this is what shouldn't happen. Formstate and component should be the same.